openshift/s2i/nginx-runtime/nginx.conf.template

```diff
@@ -60,9 +60,9 @@ http {
             root   /tmp/app/dist;
             index  index.html index.htm;
             try_files $uri /index.html;
-            gzip            on;
-            gzip_min_length 1000;
-            gzip_types      *;
+            gzip_static  always;
```

Comment:
serve the static gzip files, or decompress on demand on the server and serve js if the client doesn't support gzip.